src/playground/dto/private-bid-params.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { IsString, IsNumberString } from 'class-validator';
+import { IsOptionalString } from 'src/decorators/IsOptionalString';
+
+export class PrivateBidParams {
+  @IsString()
+  mintAccount: string;
+  @IsNumberString()
+  price: number;
+  @IsOptionalString()
```

Comment:
Why are these optional strings?Don't we always need them to have values?

If they are marked as optional on purpose, make sure to change the line below as well to
`seller?: string;`

Reply:
a bidder can give seller or tokenAccount or both for an nft they want to make a private bid on